web/app.js

```diff
@@ -1,15 +1,21 @@
-var express = require('express');
-var path = require('path');
-var bodyParser = require('body-parser');
-var index = require('./routes/index');
-var users = require('./routes/users');
 
-var app = express();
-var PORT = 3000;
+/*jshint esversion: 6 */
+
+const express = require('express');
+const path = require('path');
+const bodyParser = require('body-parser');
+const index = require('./routes/index');
+const users = require('./routes/users');
+
+const app = express();
+const http = require('http').Server(app)
+const io = require('socket.io')(http)
+const PORT = 3000;
 
 // view engine setup
 app.set('views', path.join(__dirname, 'views'));
 app.set('view engine', 'ejs');
+app.set("io", io);
```

Comment:
Any specific reason for doing this? Is `io` needed in a router? You could use a closure to pass `io` to a router.